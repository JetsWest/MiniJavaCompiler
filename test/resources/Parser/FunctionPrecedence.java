class A {
    public static void main(String[] args) {
        System.out.println(new B().Func(5));
    }

}
class B {
    public Foo Func(int q) {
        Foo a;
        Foo b;
        Foo c;
        c = a+b.Bar();
        return c;
    }
}