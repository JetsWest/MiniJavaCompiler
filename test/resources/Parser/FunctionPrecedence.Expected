Program
  MainClass A
    Print
      new B().Func(5)
  Class B
    MethodDecl Func
      returns Foo
      parameters:
        int q
      Foo a
      Foo b
      Foo c
      c = (a + b.Bar())
      Return c